### ISSUE TITLE
Cannot compile autossh on x86_64 Core m3

### ISSUE THREAD

**Issue by aLostEngineer** [skycocker/chromebrew#1211]
Like it says in the title, I cannot compile autossh. It fails during ./configure with the following message:
```
chronos@localhost / $ crew install autossh ctorrent
autossh: The purpose of autossh is to start an SSH connection, monitor it, and restart it if necessary.
http://www.harding.motd.ca/autossh
version 9c72d3
No precompiled binary available for your platform, downloading source...
######################################################################## 100.0%
Archive downloaded
Unpacking archive, this may take awhile...
gcc (GCC) 4.9.4
Copyright (C) 2015 Free Software Foundation, Inc.
This is free software; see the source for copying conditions.  There is NO
warranty; not even for MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.

Building from source, this may take a while...
checking for gcc... gcc
checking for C compiler default output file name... a.out
checking whether the C compiler works... yes
checking whether we are cross compiling... no
checking for suffix of executables... 
checking for suffix of object files... o
checking whether we are using the GNU C compiler... yes
checking whether gcc accepts -g... yes
checking for gcc option to accept ANSI C... none needed
checking for ssh... /usr/bin/ssh
checking how to run the C preprocessor... /lib/cpp
configure: error: C preprocessor "/lib/cpp" fails sanity check
See `config.log' for more details.
autossh failed to install: `./configure` exited with 1
```

Also, I cannot find config.log

(With mlocate installed)
```
$ sudo updatedb
$ locate config.log
$
```
### cpp info

```
chronos@localhost / $ cpp --version
cpp (GCC) 4.9.4
Copyright (C) 2015 Free Software Foundation, Inc.
This is free software; see the source for copying conditions.  There is NO
warranty; not even for MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
chronos@localhost / $ cpp -v
Using built-in specs.
COLLECT_GCC=cpp
Target: x86_64-cros-linux-gnu
Configured with: /src/gcc-4.9.4/configure --prefix=/usr/local --build=x86_64-linux-gnu --host=x86_64-cros-linux-gnu --target=x86_64-cros-linux-gnu --enable-languages=c,c++,fortran --disable-static --disable-werror --with-gmp-include=/opt/cross/x86_64-cros-linux-gnu/usr/local/include --with-gmp-lib=/opt/cross/x86_64-cros-linux-gnu/usr/local/lib64 --with-mpfr-include=/opt/cross/x86_64-cros-linux-gnu/usr/local/include --with-mpfr-lib=/opt/cross/x86_64-cros-linux-gnu/usr/local/lib64 --with-mpc-include=/opt/cross/x86_64-cros-linux-gnu/usr/local/include --with-mpc-lib=/opt/cross/x86_64-cros-linux-gnu/usr/local/lib64 --with-isl-include=/opt/cross/x86_64-cros-linux-gnu/usr/local/include --with-isl-lib=/opt/cross/x86_64-cros-linux-gnu/usr/local/lib64 --with-cloog-include=/opt/cross/x86_64-cros-linux-gnu/usr/local/include --with-cloog-lib=/opt/cross/x86_64-cros-linux-gnu/usr/local/lib64 --disable-multilib --disable-libmpx
Thread model: posix
gcc version 4.9.4 (GCC) 
COLLECT_GCC_OPTIONS='-E' '-v' '-mtune=generic' '-march=x86-64'
 /usr/local/libexec/gcc/x86_64-cros-linux-gnu/4.9.4/cc1 -E -quiet -v -imultiarch x86_64-linux-gnu - -mtune=generic -march=x86-64
ignoring nonexistent directory "/usr/local/include/x86_64-linux-gnu"
ignoring nonexistent directory "/usr/local/lib/gcc/x86_64-cros-linux-gnu/4.9.4/../../../../x86_64-cros-linux-gnu/include"
ignoring nonexistent directory "/usr/include/x86_64-linux-gnu"
#include "..." search starts here:
#include <...> search starts here:
 /usr/local/lib/gcc/x86_64-cros-linux-gnu/4.9.4/include
 /usr/local/include
 /usr/local/lib/gcc/x86_64-cros-linux-gnu/4.9.4/include-fixed
 /usr/include
End of search list.
```

### ASUS Chromebook Flip C302CA-DHM4
```
chronos@localhost / $ uname -a
Linux localhost 3.18.0-15946-gcf8b2a86d44d #1 SMP PREEMPT Tue Sep 5 12:34:38 PDT 2017 x86_64 Intel(R) Core(TM) m3-6Y30 CPU @ 0.90GHz GenuineIntel GNU/Linux
```

**Comment by jam7** [comment]
I guess `crew install buildessential` will solve your problem similar to #654, #952.

**Comment by aLostEngineer** [comment]
Yep, that fixed it.

Does `buildessential` need to be added to the prereqs?

**Comment by jam7** [comment]
I think so.  Others have different opinions as discussed in #952.

**Comment by cstrouse** [comment]
Fixed by #1217.
